_posts/2023-03-17-introduction-to-discrete-event-simulations-in-healthcare/introduction-to-discrete-event-simulations-in-healthcare.Rmd

```diff
@@ -0,0 +1,112 @@
+---
+title: "Introduction to Discrete Event Simulations in Healthcare"
+description: |
+  An introduction to Discrete Events Simulations (DES) in healthcare with an overview of the use of DES for modelling outpatient services appointments at Providence Healthcare.
+  
+author:
+  - name: Kevin Wang, Neil Mistry
+    url: https://www.linkedin.com/in/mingkun-wang/, https://www.linkedin.com/in/neilmistry/
+date: 2022-09-23
+output:
+  distill::distill_article:
+    self_contained: false
+draft: true
+category: post-miscellaneous
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+# Discrete Events Simulation at Providence Outpatient Care & Services Covid-19 Recovery scheduling
+
+
+Discrete Events Simulations (DES) are a method of modeling the operation of real world systems digitally.  It does this by modeling a series of events in time.  Each event is ordered by time of occurance, and the system is advanced to each event in the sequence, meanwhile updating the state of the system.  This is done so that we can test hypothetical changes to the system and their impacts on metrics we are interested in, uncover bottlenecks or process challenges and make overall assessments of the system.
```

Comment:
```suggestion
Discrete Events Simulations (DES) are a method of modeling the operation of real world systems digitally.  It does this by modeling a series of events in time.  Each event is ordered by time of occurrence, and the system is advanced to each event in the sequence, meanwhile updating the state of the system.  This is done so that we can test hypothetical changes to the system and their impacts on metrics we are interested in, uncover bottlenecks or process challenges and make overall assessments of the system.
```

Reply:
Maybe `Each event is ordered by time of occurrence, and the system is advanced to each event in the sequence, meanwhile updating the state of the system.` could be re-worded to something a bit more clear?

Some suggestions:
- Events are ordered by time of occurrence, and the DES system updates its state as it traverses through each event.
- The DES updates the state of the system as it steps through each event in time.
- Series of events are ordered by time of occurrence. The system then steps through each event and updates its state.
- The system steps through each event in the time-ordered sequence and updates its state.